fuse/src/main/java/tachyon/fuse/TachyonFuse.java

```diff
@@ -0,0 +1,152 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.fuse;
+
+import java.nio.file.Paths;
+import java.util.List;
+
+import com.google.common.collect.Lists;
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.CommandLineParser;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.HelpFormatter;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+import org.apache.commons.cli.ParseException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import tachyon.Constants;
+import tachyon.client.ClientContext;
+import tachyon.conf.TachyonConf;
+
+/**
+ * Main entry point to Tachyon-FUSE
+ */
+public final class TachyonFuse {
+  private static final Logger LOG = LoggerFactory.getLogger(Constants.LOGGER_TYPE);
+  private static TachyonConf sTachyonConf;
+
+  public static void main(String[] args) {
+    sTachyonConf = ClientContext.getConf();
+    final TachyonFuseOptions opts = parseOptions(args);
+    if (opts == null) {
+      System.exit(1);
+    }
+
+    final TachyonFuseFs fs = new TachyonFuseFs(sTachyonConf, opts);
+    final List<String> fuseOpts = opts.getFuseOpts();
+    fuseOpts.add("-odirect_io");
+
+    try {
+      fs.mount(Paths.get(opts.getMountPoint()), true, opts.isDebug(),
+          fuseOpts.toArray(new String[0]));
+    } finally {
+      fs.umount();
+    }
+  }
+
+  /**
+   * Parses CLI options
+   * @param args CLI args
+   * @return Tachyon-FUSE configuration options
+   */
+  private static TachyonFuseOptions parseOptions(String[] args) {
+    final Options opts = new Options();
+    final Option mntPoint = Option.builder("m")
+        .hasArg()
+        .required(false)
+        .longOpt("mount-point")
+        .desc("Path where tachyon-fuse should be mounted.")
```

Comment:
is my understanding correct? flag `mount-point` is a path in local file system people want to mount Tachyon to, and the following flag `tachyon-root` is a path within Tachyon file system space that will show up in `mount-pioint`? Probably a little bit more details in desc here and below.